docs/topics/native/native-spm.md

```diff
@@ -0,0 +1,192 @@
+[//]: # (title: Swift Package export setup)
+
+Kotlin/Native output for an Apple target can be consumed as an SPM dependency.
+For example, if your Kotlin Multiplatform project has an iOS target, you may want to make the binary available
+as a dependency to iOS developers working on native Swift projects in Xcode. Using KMP tooling, you can provide
+an artifact that would seamlessly integrate with Xcode experience.
+
+This guide shows how to do this using the [XCFramework building](multiplatform-build-native-binaries.md#build-xcframeworks)
+capability of Kotlin Gradle.
+
+## Prepare locations for the files to be uploaded
+
+To make your framework available to be consumed, you will need to upload two files:
+* A ZIP archive of the XCFramework itself should be uploaded to a convenient file storage with direct access (for example,
+S3, GitHub Releases, or a Maven repository). Choose the option that is easiest to integrate into your workflow.
+* The `Package.swift` file describing the package needs to be placed in a Git repository. You need to decide which
+repository to use. Pros and cons for several options are discussed below.
+
+### Options for Package.swift distribution
+The `Package.swift` that describes the Swift Package needs to be placed in a Git repository:
+* Store the `Package.swift` file in an independent repository to make its versioning separate from the
```

Comment:
Again, I'd recommend to highlight the options and split the sentences. This is what I came up with:

> * Store the Package.swift file in an independent repository. It helps to separate its versioning from the Kotlin Multiplatform project. This is the recommended approach. It's scalable and generally easier to maintain.
> * Put the Package.swift file in the same repository as your Kotlin Multiplatform code. It's a more straightforward approach, but keep in mind that, in this case, a Swift package and the code have the same versioning. SPM uses Git tags to version packages, which can conflict with tags used for the project.
> * Store the Package.swift within the consumer project's repository. It helps to avoid versioning and maintenance issues. However, this approach can cause problems with multi-repository SPM setups of the consumer project and further automation: